summary_workflows/quantification/quantification_dockers/q_consolidation/manage_assessment_data.py

```diff
@@ -121,7 +207,10 @@ def generate_manifest(data_dir,output_dir, participant_data):
             elif metrics_data["metrics"]["metric_id"] == metric_Y:
```

Comment:
lines 186-198 I couldn't figure out. Don't know which file is read here and whether it will ever exist, I couldn't find the key `challenge_type` anywhere.

Reply:
I suggest we remove this code as I don't think it ever runs with the current directory structure. And even if it ran, it seems that the content added to the `challenge_participants` list in [rows 194-198](https://github.com/iRNA-COSI/APAeval/blob/07b96d7f013a4aa0241bc4ad35772329b02cb92e/summary_workflows/quantification/quantification_dockers/q_consolidation/manage_assessment_data.py#L194-LL198) would never make it into the aggregation file since I don't see it being referenced again.

Reply:
Correction to myself: the `challenge_participants.append()` in row 194 would actually change the content of `challenge_participants` in `aggregation_file` since arrays are passed as reference.

Reply:
Anyway, I still think we can ignore it since that doesn't change what I said about this snippet not  running due to the current directory structure.